### PR TITLE
Harrison/parameter

### PR DESCRIPTION
**Pull request by hwchase17**
add parameter to use original question or not

### CODE REVIEW

File: langchain/chains/conversational_retrieval/base.py [hunk 1]
```diff
@@ -63,6 +63,11 @@ class BaseConversationalRetrievalChain(Chain):
     a new standalone question to be used later on."""
     output_key: str = "answer"
     """The output key to return the final answer of this chain in."""
+    use_new_question: bool = True
```
Comment (nfcampos) [inline langchain/chains/conversational_retrieval/base.py:66]: Is `use` clear enough?

File: langchain/chains/conversational_retrieval/base.py [hunk 2]
```diff
@@ -132,7 +137,8 @@ def _call(
         else:
             docs = self._get_docs(new_question, inputs)  # type: ignore[call-arg]
         new_inputs = inputs.copy()
-        new_inputs["question"] = new_question
+        if self.use_new_question:
+            new_inputs["question"] = new_question
```
Comment (baskaryan) [inline langchain/chains/conversational_retrieval/base.py:141]: should we not generate the new question at all in this case

Comment (hwchase17) [reply langchain/chains/conversational_retrieval/base.py:140]: need it for retrieval

File: langchain/chains/conversational_retrieval/base.py (hunk 1 above)
Comment (hwchase17) [reply langchain/chains/conversational_retrieval/base.py:66]: any suggestions?

**Pull request merged**